scripts/release-manager.js

```diff
@@ -1,248 +0,0 @@
-    // get the contents of an existing gh issue by id
-    // or it will default to the most recent one by label
-    // this is so it will preserve state of checked todo items
-    : await ghIssue({
-      ...args,
-      command: typeof args.update === 'string' ? 'view' : 'list',
-      number: args.update,
-    })
-
-  return ghIssue({
-    ...args,
-    command: number ? 'edit' : 'create',
-    number,
-    body: replace(body),
-    title: replace(title),
-  })
-}
-
-const parseArgs = raw => {
-  const result = {
-    create: false,
-    update: null,
-    repo: 'npm/cli',
-    label: 'release: manager',
-    assignee: '@me',
-    date: date(),
-    version: 'X.Y.Z',
-    // look for that heading level with a match for the portion after
-    section: '### .*cli.*',
-    release:
-      'https://raw.githubusercontent.com/wiki/npm/cli/Release-Process-(v8).md',
-  }
-
-  const replacements = {}
-
-  const clean = {
-    // this script will not work correctly with the tag style
-    // of the version (prefixed with a v) so strip it out
-    version: v => v.replace(/^v/g, ''),
-  }
-
-  const shorts = {
-    R: 'repo',
-    l: 'label',
-    a: 'assignee',
-    d: 'date',
-    v: 'version',
-    c: 'create',
-    u: 'update',
-  }
-
-  const camel = k => k.replace(/-([a-z])/g, a => a[1].toUpperCase())
-
-  // parse argv into array of [k,v] pairs
-  // works with --x=1 --x 1 --x -x
-  const argv = raw
-    .join(' ') // join to a string
-    .split(/(?:^|\s+)-/g) // split on starting dashses
-    .map(x => x.trim().replace(/\s+/g, ' ')) // collapse spaces
-    .filter(Boolean) // remove empties
-    .map(x => x.split(/[=\s]/)) // split on equal or space
-    .map(([k, v]) => [
-      // we split on the initial dash previously so now
-      // 1 dash means 2 and 0 means 1
-      ...(k.startsWith('-') ? ['--', k.slice(1)] : ['-', k]),
-      v ?? true, // default to true for no value
-    ])
-    .map(([dash, key, value]) => ({ dash, key: camel(key), value }))
-
-  for (const { dash, key, value } of argv) {
-    const k = dash.length < 2 ? shorts[key] : key
-    if (Object.hasOwn(result, k)) {
-      result[k] = clean[k] ? clean[k](value) : value
-    } else {
-      // any unknown arg is a replacement value
-      replacements[k] = value
-    }
-  }
-
-  if (!result.create && !result.update) {
-    // set default to create if no command is specified
-    result.create = true
-  } else if (result.create && result.update) {
-    throw new Error('Cannot set both create and update')
-  }
-
-  if (result.help) {
-    console.error(usage())
-    return process.exit(0)
-  }
-
-  return {
-    ...result,
-    replacements: {
-      '(\\d+\\.\\d+\\.\\d+|X\\.Y\\.Z)': result.version,
-      '(\\d{4}-\\d{2}-\\d{2}|YYYY-MM-DD)': result.date,
-      ...replacements,
-    },
-  }
-}
-
-main(parseArgs(process.argv.slice(2)))
-  .then(d => console.log(d))
-  .catch(err => {
-    console.error(err)
-    process.exitCode = 1
-  })
```

Comment:
This goes away too since `release-please` handles all of our changelog for us now.